SQL/Archive/19.2/2018-02-27_normalize_mri_protocol.sql

```diff
@@ -0,0 +1,31 @@
+-- This patch adds min & max columns for every field in `mri_protocol` which presently can hold range values. 
+ALTER TABLE
+	`mri_protocol`
+		ADD COLUMN `TR_min` varchar(10) DEFAULT NULL,
+		ADD COLUMN `TR_max` varchar(10) DEFAULT NULL,
+		ADD COLUMN `TE_min` varchar(10) DEFAULT NULL,
+		ADD COLUMN `TE_max` varchar(10) DEFAULT NULL,
+		ADD COLUMN `TI_min` varchar(10) DEFAULT NULL,
+		ADD COLUMN `TI_max` varchar(10) DEFAULT NULL,
+		ADD COLUMN `slice_thickness_min` varchar(10) DEFAULT NULL,
+		ADD COLUMN `slice_thickness_max` varchar(10) DEFAULT NULL,
+		ADD COLUMN `FOV_x_min` varchar(5) DEFAULT NULL,
+		ADD COLUMN `FOV_x_max` varchar(5) DEFAULT NULL,
+		ADD COLUMN `FOV_y_min` varchar(5) DEFAULT NULL,
+		ADD COLUMN `FOV_y_max` varchar(5) DEFAULT NULL,
+		ADD COLUMN `FOV_z_min` varchar(5) DEFAULT NULL,
+		ADD COLUMN `FOV_z_max` varchar(5) DEFAULT NULL,
+		ADD COLUMN `xspace_min` varchar(5) DEFAULT NULL,
+		ADD COLUMN `xspace_max` varchar(5) DEFAULT NULL,
+		ADD COLUMN `yspace_min` varchar(5) DEFAULT NULL,
+		ADD COLUMN `yspace_max` varchar(5) DEFAULT NULL,
+		ADD COLUMN `zspace_min` varchar(5) DEFAULT NULL,
+		ADD COLUMN `zspace_max` varchar(5) DEFAULT NULL,
+		ADD COLUMN `xstep_min` varchar(5) DEFAULT NULL,
+		ADD COLUMN `xstep_max` varchar(5) DEFAULT NULL,
+                ADD COLUMN `ystep_min` varchar(5) DEFAULT NULL, 
+                ADD COLUMN `ystep_max` varchar(5) DEFAULT NULL,
+                ADD COLUMN `zstep_min` varchar(5) DEFAULT NULL,
+                ADD COLUMN `zstep_max` varchar(5) DEFAULT NULL,
+		ADD COLUMN `time_min` varchar(50) DEFAULT NULL,
+		ADD COLUMN `time_max` varchar(50) DEFAULT NULL;
```

Comment:
should contain "AFTER" clause to they be inserted before the "series_description_regex" column. This would keep consistant with creation of table from scatch (0000-00-00-schema.sql). Specially if min and max are converted to numerical type instead of varchar().